### ISSUE TITLE
Some Sources Are Broken

### ISSUE THREAD

**Issue by faizygr8** [coder-alpha/FMoviesPlus.bundle#60]
Hi Coder-Alpha, Thanks for Maintaining the Plugin.

Flixanity and Primwire Source are Broken, Can you Please fix Them?
Primewire Mainly as it has more Content.
Thanks Mate

**Comment by zablon123** [comment]
Primewire hasn't worked for a long time since the site went down...is it back up somewhere?

**Comment by faizygr8** [comment]
> 
> 
> Primewire hasn't worked for a long time since the site went down...is it back up somewhere?

I have Been Using This With no Issues
https://primewire.unblocked.llc/

I tried Changing the URL MySelf but it does not Help as i am no Coding Expert.

**Comment by zablon123** [comment]
Cool. Didn't know it was back. The Primewire channel is still out there, but with plugins going away soon I doubt anyone will try to update it. I might play around and see if just changing the urls makes it work, but my guess is the underlying code/links changed too.

**Comment by zablon123** [comment]
So, the Primewire plugin does work if you put in https://www.letmewatchthis.cx  - which is another primewire mirror. At the moment it can't find sources, but that should just be a matter of a few line changes if i can figure them out.

**Comment by faizygr8** [comment]
Thanks Pal for your Efforts. It Would be Great if You Can Figure it out. Thanks

**Comment by faizygr8** [comment]
Thanks Pal is it Scrapping the Sources?

**Comment by MeanLauraJean** [comment]
Hey @zablon123 I changed the URL in primewire_my_tv.py to https://letmewatchthis.cx but it doesn't seem to work. I can see the updated URL under External Providers however search results for any movie that's available on the letmewatchthis.cx site don't appear using the FMovies plugin. Is it working for you?

**Comment by zablon123** [comment]
> 
> 
> Hey @zablon123 I changed the URL in primewire_my_tv.py to https://letmewatchthis.cx but it doesn't seem to work. I can see the updated URL under External Providers however search results for any movie that's available on the letmewatchthis.cx site don't appear using the FMovies plugin. Is it working for you?
> 
> For now I've moved primewire_my_tv.py to the disabled folder since it's not working (even before the URL change).

No I don't believe it is. The site works as I was able to test it in Primewire app on Roku and you can navigate and see what is there, but it doesn't seem to be scraping any links and either app. In both Fmovies and Primewire I think someone would need to update the scraping code to get them to work. It's a bit beyond my abilities. It may be something simple like a URL format change, and I looked for something like that but I was unable to get it to work.

**Comment by coder-alpha** [comment]
@zablon123 @faizygr8 @MeanLauraJean 
New ver. 0.74 is up. Addresses fixes to Primewire and a few other non-working Sources. Please create a new issue for specific source if issues persist. Thanks.
